packages/react/src/card/card.tsx

```diff
@@ -204,16 +204,27 @@ export interface CardProps extends CardBaseProps {
    * @default "lighter-brand"
    * */
   color?: "white" | "lighter-brand" | "light-grey-fill";
+  /**
+   * On cards that can have images to the left or right of the text, (fullwidth or focus),
+   * left position is default. Setting imageRight to true, puts the image to the right.
+   *
+   * @default false
+   * */
+  imageRight?: boolean;
```

Comment:
union props are often a better choice than boolean ones. Makes it more explicit as well
```suggestion
   * @default "left"
   * */
  imageRight?: "left" | "right";
```

https://www.selbekk.io/blog/the-10-component-commandments

<img width="910" alt="image" src="https://github.com/user-attachments/assets/621dbcfe-c5ba-4b8b-897a-495abd87f5b2" />

Reply:
Arguing with myself is a nice way to learn ways of thinking: 

Isn't this an example of a truly-binary choice?

You could say that if more image-position options than just left/right are added at some point (eg. "top-left") then it won't be binary anymore. 

Or that if we at some point add an `imageLeft` prop as well (to control something else in the card, since making it control the same image would make no sense), then that would appear from the outside to contradict and compete with `imageRight`, and be confusing. 

But at the same time, when changing things like in those examples, you'd be altering the type structure anyway, and might as well change from the boolean _then_, instead of always spending resources on strict futureproofing/premature optimizing. 

But I guess the difference is preventing the breaking signature change: consuming components would also need to be changed.

Fair enough, will do. :neckbeard: 😁